packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -23,20 +24,23 @@ import 'tabs.dart';
 import 'text_theme.dart';
 import 'theme.dart';
 
-const double _kLeadingWidth = kToolbarHeight; // So the leading button is square.
+const double _kLeadingWidth =
+    kToolbarHeight; // So the leading button is square.
```

Comment:
flutter style guide says to keep these on one line, even if htye go over 80 chars.